app/services/rapid_ready_for_decision/hypertension_special_issue_manager.rb

```diff
@@ -15,6 +15,8 @@ def add_special_issue
         add_rrd_code(disability) if hypertension_increase?(disability)
       end
       submission.update!(form_json: JSON.dump(submission_data))
+      submission.invalidate_form_hash
+      submission
```

Comment:
It's not clear that this method wants/needs to return a `submission` record, but let's do that for consistency with `hypertension_upload_manager.rb`.